css/admin.upgrade.css

```
@@ -84,19 +92,23 @@
 	text-decoration: none;
 	transition-duration: .2sec;
 }
+
 .genesis-custom-blocks-pro .btn:hover {
 	cursor: pointer;
 	background-color: var(--color-white);
 	color: var(--color-brand);
 }
+
 .genesis-custom-blocks-pro .btn-secondary {
```

```suggestion
.genesis-custom-blocks-pro .container .btn-secondary {
```

This will help fix the styling of the two lower buttons.